pycode/memilio-epidata/memilio/epidata/getDIVIData.py

```diff
@@ -56,8 +56,6 @@ def fetch_divi_data(
 ) -> pd.DataFrame:
     """! Downloads or reads the DIVI ICU data and writes them in different files.
 
-    Available data starts from 2020-04-24.
-    If the given start_date is earlier, it is changed to this date and a warning is printed.
```

Comment:
Why not keeping these two lines?

Reply:
I wanted to add something for the end date and realized that here is not described what is really done in the function.